MVP Module/Source/Feed Presentation/FeedLoadingErrorViewModel.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  FeedLoadingErrorViewModel.swift
+//  MVP
+//
+//  Created by Ksenia on 21.07.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+struct FeedLoadingErrorViewModel {
```

Comment:
The concept of `loading` and `error` should be separate since `errors` are not just for loading errors. We can represent errors for many other operations. I suggest renaming this ViewModel:
```suggestion
struct FeedErrorViewModel {
```